initramfs/src/main.rs

```diff
@@ -0,0 +1,44 @@
+use clap::Parser;
+use env_logger::Env;
+use log::{debug, info};
+
+mod filesystem;
+mod image;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    #[arg(short, long)]
+    image: String,
+}
+
+#[tokio::main]
+async fn main() {
```

Comment:
Your main should return a proper Result, and avoid the following `unwrap`s.

Reply:
I fixed this while force pushing for the last time